tools/cldr-apps/src/main/webapp/js/CldrSurveyVettingLoader.js

```diff
@@ -650,7 +661,7 @@ function showV() {
 
 						{
 							title: 'Admin Panel',
-							url: surveyUserURL.adminPanel,
+							url: 'SurveyAjax?what=admin_panel&s=' + sessionId,
 							display: (surveyUser && surveyUser.userlevelName === 'ADMIN')
```

Comment:
This url should be moved into surveyUserURL, for consistency -- note that it no longer needs "vap"